src/modules/esl-media/core/esl-media-iobserver.ts

```diff
@@ -27,11 +27,13 @@ function handleViewport(entry: IntersectionObserverEntry): void {
   }
 
   // Videos that playing and out of min ratio RATIO_TO_STOP should be stopped
-  if (video.active && entry.intersectionRatio <= RATIO_TO_STOP) {
-    video.pause();
+  if (entry.intersectionRatio <= RATIO_TO_STOP) {
+    if (video.active) video.pause();
+    video.$$attr('in-viewport', false);
```

Comment:
The biggest problem now is that we need to revisit this logic.
It doesn't sound correct that we affect component twicely: more indirect - attribute, directly - action (play/pause)
In case we going to maintain indirect state it sounds like we need to move activation logic on esl-media canPlay level.

Reply:
> The biggest problem now is that we need to revisit this logic. It doesn't sound correct that we affect component twicely: more indirect - attribute, directly - action (play/pause) In case we going to maintain indirect state it sounds like we need to move activation logic on esl-media canPlay level.

Now we introduced way too many switch cases
`  protected override attributeChangedCallback(attrName: string, oldVal: string, newVal: string): void {
    if (!this.connected || oldVal === newVal) return;
    // eslint-disable-next-line sonarjs/max-switch-cases`